src/main/proto/ga4gh/allele_annotation_service.proto

```diff
@@ -145,3 +148,81 @@ message GetVariantAnnotationSetRequest {
   // The ID of the `VariantAnnotationSet` to be retrieved.
   string variant_annotation_set_id = 1;
 }
+
+/******************  /transcripteffects  *********************/
+// This request maps to the body of `POST /transcripteffects/search` as
+// JSON.
+message SearchTranscriptEffectsRequest {
+  // Required. The `VariantAnnotationSet` to search.
+  string variant_annotation_set_id = 1;
+  
+  // If a variant_annotation_id is provided only return transcript effects
+  // from this annotation.
+  // Either variant_annotation_id feature_id, or range is required.
+  string variant_annotation_id = 2;
+
+  // Only return variants with reference alleles on the reference with this
+  // name. One of this field or `reference_id` is required if specifying a range.
+  string reference_name = 3;
+
+  // Only return variants with reference alleles on the reference with this
+  // ID. One of this field or `reference_name` is required if specifying a range.
+  string reference_id = 4;
+
+  // Required if specifying a range. The beginning of the
+  // window (0-based, inclusive) for which variants with overlapping reference
+  // alleles should be returned. Genomic positions are non-negative integers
+  // less than reference length. Requests spanning the join of circular
+  // genomes are represented as two requests one on each side of the join
+  // (position 0).
+  int64 start = 5;
```

Comment:
Leaning towards keeping the sign based on the response to https://github.com/ga4gh/schemas/issues/737